presto-hive/src/main/java/com/facebook/presto/hive/HiveClientModule.java

```diff
@@ -95,6 +96,7 @@ public void configure(Binder binder)
         recordCursorProviderBinder.addBinding().to(ColumnarBinaryHiveRecordCursorProvider.class).in(Scopes.SINGLETON);
         recordCursorProviderBinder.addBinding().to(GenericHiveRecordCursorProvider.class).in(Scopes.SINGLETON);
 
+        binder.bind(EventClient.class).to(HiveEventClient.class).in(Scopes.SINGLETON);
```

Comment:
Instead of doing this, install `EventModule`, then add a binding here:
```java
newSetBinder(binder, EventClient.class).addBinding().to(HiveEventClient.class).in(Scopes.SINGLETON);
```
The event module has a multi-event client that sends the events to all bound clients, allowing multiple event listeners